CRM/Admin/Form/Options.php

```diff
@@ -376,10 +376,36 @@ public static function formRule($fields, $files, $self) {
       }
     }
 
+    $optionGroup = $this->getOptionGroupDataType($self->_gName);
```

Comment:
I think `$this` call need to be changed to `self::` under static formRule() function.